Configure

```diff
@@ -919,8 +919,17 @@ $target{dso_extension}=$target{shared_extension_simple};
 $config{cross_compile_prefix} = $ENV{'CROSS_COMPILE'}
     if $config{cross_compile_prefix} eq "";
 
+# This is textbook from the perlvar manual.  It makes sure we have
+# a perl path that can be used as a file name, or as the case may be,
+# as something that can be used with the hashbang script syntax.
+use Config;
+my $secure_perl_path = $Config{perlpath};
```

Comment:
I don't follow. perlvar manual refers  to $^X at this point, not to $Config{perlpath}. And I for one would find $^X more appropriate. Imagine that you want to test perl without installing it. You manipulate some environment variables and off it goes. And what happens if you install ActivePerl in non-default location? Some other Windows perl when you have option to alter target installation directory? I have deja-vu feeling as if I was already saying these words about Windows...